sound/soc/intel/boards/bxt_pcm512x.c

```diff
@@ -166,6 +166,12 @@ static const struct snd_soc_ops aif1_ops = {
 	.shutdown = aif1_shutdown,
 };
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_VIRTIO_BE)
+#define SSP5_NAME "vm_dai_link"
+#else
```

Comment:
We cant do this (conditional compillation), I assume this is for Guests ? GOS machine driver will be very generic and use topology to defne BE and FE DAI links. SOS machine drivers wont change at all.

Reply:
@lgirdwood no, actually that's for the host. "BE" is Back End, i.e. the host / SOS. I have to clean up all the special names that are used throughout the patch set like this one, also "VM FE Playback", "VM FE Capture", "VIO", "vm_fe_playback", "vm_fe_capture" - I have to find a minimum and meaningful set of those

Reply:
No, this was actually for the host. On one hand, this specific hunk is for the bxt-pcm512x machine driver and as such, this is mostly an example (this will also be a separate patch). On the other hand - yes, this string matching and hard-coded names across the code are ugly. Unfortunately I haven't got the time yet to rework them. Each of them has to be considered and a better solution has to be found. This is one of the present limitations of the patch set.